### ISSUE TITLE
ember-cli v0.2.0: The package `ember-data` is not a properly formatted package

### ISSUE THREAD

**Issue by visualjeff** [emberjs/data#2846]
Any idea why the following message shows up when using ember 0.2.0?

The package ember-data is not a properly formatted package, we have used a fallback lookup to resolve it at /CoolThing/node_modules/ember-data. This is generally caused by an addon not having a main entry point (or index.js).

I'm running ember-cli v0.2.0 within a docker container. Its when I ran the docker container, created a new ember app then attempted to build it that I noticed this message.

**Comment by bmac** [comment]
The currently released version of Ember Data is missing a main entry in its `package.json` @rwjblue opened a pr yesterday to fix this https://github.com/emberjs/data/pull/2842 issue. When we release a beta.16, hopefully soon, this warning will go away.

**Comment by rwjblue** [comment]
I addressed this question in https://github.com/ember-cli/ember-cli/issues/3477 (much the same as @bmac), not sure why you opened two issues for this...

**Comment by visualjeff** [comment]
Sorry about the dup.

**Comment by rwjblue** [comment]
Nah, sorry for being a jerk. :blush: 

We should leave both open in case folks look in either place when they see the warning...

**Comment by esbanarango** [comment]
:+1:

**Comment by pixelmint** [comment]
Is this going to affect ember-data in any way or is this just a warning?

**Comment by rwjblue** [comment]
Warning only

**Comment by selbyk** [comment]
Definitely startled me today after recloning a repo. Glad it's nothing to worry about. Really looking forward to the up coming releases.  Keep up the good work. ^.^

**Comment by laantorchaweb** [comment]
I don't know if this is related but I was updating a current project to ember-cli 2.0 and I got this warning, but I can't execute ember anymore. I'm on a windows machine and this is the full error.

```
The package `ember-data` is not a properly formatted package, we have used a fallback lookup to resolve it at `c:\projects\uptop_cms\node_modules\ember-data`. This is generally caused by an addon not having a `main` entry point (or `index.js`).
%1 is not a valid Win32 application.
c:\MYPROJECT\node_modules\ember-cli-sass\node_modules\broccoli-sass\node_modules\node-sass\vendor\win32-x64-node-0.10\binding.node
```

**Comment by simo163** [comment]
It seems to be only a warning, wish this message will go away soon. It's always nice to have all green :)

**Comment by Ba5ik7** [comment]
Thanks, my heart dropped there for a sec.

**Comment by twokul** [comment]
seems like we can close this one in favour for ember-cli/ember-cli#3477

**Comment by bmac** [comment]
Closing this should be fixed in Ember Data 1.0.0-beta.16
